docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -232,4 +330,19 @@ GET books/_search
   }
 }
 ----
-// TEST[continued]
\ No newline at end of file
+// TEST[continued]
+
+[discrete]
+[[getting-started-delete-indices]]
+=== Delete index
+
+Delete indices using the <<indices-delete-index,`DELETE` index API>>.
+
+[source,console]
+----
+DELETE /books
+DELETE /my-explicit-mappings-books
+----
+// TEST[skip:handled by setup/teardown]
+
+
```

Comment:
I don't know how this works but assume the "setup elasticsearch" at the bottom right of the article is there because of the next node in the ToC?